ide/che-core-ide-templates/src/main/resources/samples.json

```diff
@@ -1564,5 +1564,28 @@
     "tags": [
       "TypeScript","ts"
     ]
+  },
+  {
+    "name": "ceylon-hello-world",
+    "displayName": "ceylon-hello-world",
+    "path": "/ceylon-hello-world",
+    "description": "A Ceylon 'Hello World' project example that runs on all supported backends.",
+    "projectType": "blank",
+    "mixins": [],
+    "attributes": {
+    },
+    "modules": [],
+    "problems": [],
+    "source": {
+      "type": "git",
+      "location": "https://github.com/davidfestal/ceylon-hello-world",
```

Comment:
would be great if it could. But @TylerJewell is the only member of this organization afaik.
OTH there are already quite a few samples whose git URL is from seom other place: https://github.com/tomitribe, https://github.com/kaloyan-raev, bitnami for example